api/.env.defaults

```diff
@@ -1,6 +1,6 @@
 NODE_CONFIG_DIR=../config
 ALLOW_CONFIG_MUTATIONS=1
-NODE_CONFIG_ENV=development
+NODE_CONFIG_ENV=demo
```

Comment:
Can this be reverted?

Reply:
@udaydenduluri33 - looks like this got changed via PR #2886 for EUI-7858; I'll revert it